com.github.opentyrian.OpenTyrian.yaml

```diff
@@ -0,0 +1,53 @@
+app-id: com.github.opentyrian.OpenTyrian
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --device=all
+  - --share=network
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Pulseaudio access
+  - --socket=pulseaudio
+  #SDL needs to be able to talk to UPower
+  - --system-talk-name=org.freedesktop.UPower
+  #Allow SDL to inhibit the screensaver
+  - --talk-name=org.freedesktop.ScreenSaver
```

Comment:
```suggestion
  - --talk-name=org.freedesktop.ScreenSaver
  - - --persist=.opentyrian
```

to hopefully fix

> warning: failed to open 'tyrian.cfg': No such file or directory

> Invalid or missing TYRIAN.CFG! Continuing using defaults.

> warning: failed to open 'opentyrian.cfg': No such file or directory
> warning: failed to open 'tyrian.sav': No such file or directory

Reply:
I don't have this problem...

Of course, after the first start I have the error. But after another start there are no more.

Reply:
And I don't have the audio error either... (manjaro linux)

Reply:
I assume you installed disabling the sandboxing. I use @openSUSE Leap, but I doubt it matters.

Reply:
I've installed it with
`flatpak install --user https://dl.flathub.org/build-repo/89216/com.github.opentyrian.OpenTyrian.flatpakref`

Reply:
That's what I did. Nonetheless, https://docs.flatpak.org/en/latest/sandbox-permissions.html?highlight=persist#filesystem-access is required here.

Reply:
bot, build com.github.opentyrian.OpenTyrian